core/src/main/java/jenkins/security/ExceptionTranslationFilter.java

```diff
@@ -235,4 +238,20 @@ public void init(FilterConfig filterConfig) throws ServletException {
 
     public void destroy() {
     }
-}
\ No newline at end of file
+
+	private AcegiSecurityException containsAccessException(Exception exception) {
```

Comment:
unused now?

Reply:
Good catch. Removed. Thanks for the review!